docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,238 @@
+= Integrating CMOS with Existing Monitoring Stacks
+:icons: font
+:imagesdir: ../assets/images
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+[abstract]
+You can integrate CMOS with your existing Prometheus, Alertmanager, and Grafana monitoring system. 
+This tutorial documents how to configure each component in order to achieve this.
+
+include::partial$tutorial.adoc[]
+
+== Pre-requisites
+
+The environment you wish to integrate CMOS into must have:
+
+. At least one **Couchbase Server** node;
+. **Prometheus**, with **Alertmanager** installed;
+. **Grafana**, configured to use the Prometheus server as a Data Source;
+
+==== Optional ====
+
+- **Docker**, if using the Cluster Monitor's Docker image instead of the binary.
+
+== Installation
+
+=== Install Cluster Monitor
+
+The Cluster Monitor can be installed by downloading a Docker image.
+
+[NOTE]
+====
+We plan for the image to be distributed on Docker Hub, but currently this is not available. 
+====
+
+Run the Docker container, using the preview image given to you by Couchbase:
+[source, console]
+----
+export CB_MULTI_ADMIN_USER=admin
+export CB_MULTI_ADMIN_PASSWORD=password
+
+docker run -d --rm \
+  -p 7196:7196 \ <.>
+  --network cmos \
+  --couchbase-user Administrator \
+  --couchbase-password password \
```

Comment:
These can be removed, because you're passing `--couchbase-user` to `docker run` (since they're before the image name). Also not sure if you need `-e` to pass through the env vars, or will it just pick it up from the host system?